### DISCUSSION TITLE
Experience with Gira Senic "Friends of Hue" switches?

### DISCUSSION THREAD

**Discussion by shypike** [Koenkk/zigbee2mqtt#15984] [Devices]
Does anyone have experience with Gira Senic "Friends of Hue" switches?
I know that they not in the database yet, but I wanted to change that.
The switches are "green", based on EnOcean electronics.
However, when I try pairing, I see no Zigbee traffic at all.
Pairing is supposed to start after pressing lower right for 10 seconds and then releasing it.
I did that several times, on three different switches.
Any tips?

**Comment by corvy** [comment]
What channel do you use?

**Reply by shypike** [reply]
Channel 11.
But I sent the switches back. Some more googling tells me that Zigbee Green Power is challenging, because it  deviates from the standard. Some people had success with different software than zigbee2mqtt, but I don’t want to switch:

**Reply by corvy** [reply]
Cool. Just in case someone else reads this later, pairing with these switches is done by pressing the top right button (for channel 11). At least if they are based on PTM 215Z. I have 3 of these and they paired and worked fine without any Hue devices. I just had Namron dimmers in addition to the switches. 

Read the channel to button maps here: https://www.zigbee2mqtt.io/devices/PTM_215Z.html#enocean-ptm%2520215z

**Comment by Danit2** [comment]
I have a Feller Friends of Hue Switch in my system.
They must nearly the same. Can you tell us witch kinde of switch you have? PTM 215Z, PTM 215ZE or PTM 216Z?
Normaly there is a sticker someware on the device. 

It is a little bit difficult to pair this device but it works.
But do you have some HUE device near the switch? Beqause this is needit to translate the communication of the switch.

**Reply by shypike** [reply]
Thanks for the suggestions. But as I said earlier, I sent the switches back to the supplier.
Regarding Hue, yes I have Hue devices, but they’re all connected to a Hue hub. My switches should connect to my home assistent box (Deconz stick and zigbee2mqtt). Using the new switches over Hue is not an option.

**Comment by ErikSteiner** [comment]
Hey there, I recently bought several `Senic Gira Friends of Hue Smart Switches`. They are actually enOcean `PTM215Z`. I have no Hue device, nor any Philips device in my Smart Home. However, I have a ZigBee light by IKEA, that works as a translator to the Green Power protocol, that is used by the switch. Without a device, that understands the Green Power protocol, you will not be able to connect the switch to your network.

The pairing of the switch was very easy. I made sure, that my ZigBee network uses a default channel. In my case `11`. The channel must be selected, by pressing the corresponding button on the switch. Afterwards the switch appears in the device tab of Zigbee2MQTT. Then the channel needs to be approved:

> To pair it hold the corresponding button for that channel for 7 seconds or more.
>
> | Button | Channel |
> | ----- | --------|
> | A0 | 15 |
> | A1 | 20 |
> | B0 | 11 |
> | B1 | 25 |
>
> Once the device is paired you need to confirm the channel. To do this press A1 and B0 together. Important: don't press any other buttons between this and the pairing.

If it is relevant, I use a `Sonoff Zigbee 3.0 USB-Dongle Plus`, with coordinator version `20240710`, on Zigbee2MQTT.

**Comment by kopierschnitte** [comment]
Unfortunately, I can't confirm this. I tried to pair two of these buttons to a vanilla installation (1st device / using CC2652P7 as coordinator) and it never got recognized. Also tried different channels and factory reset. However, pairing to an (already established and well populated) network using Deconz/Phoscon worked immediately and repeatable.

Therefore I suspect that might be the need for other (non-ZGP) devices in the network in order to pair those switches...

**Reply by ErikSteiner** [reply]
My coordinator, and I guess most coordinators don't understand the ZigBee Green Power protocol. That is used on most ultra-low power devices. In order to connect these devices, you will need a device that understands Green Power and that interacts as a translator to the ZigBee network. I have an IKEA ZigBee light in my network, that I used to pair the Senic switch. Most Hue lamps should work, too.

**Reply by kopierschnitte** [reply]
Thanks for confirmation 👍

**Reply by chris-1243** [reply]
The documentation should help you

https://www.zigbee2mqtt.io/advanced/zigbee/01_zigbee_network.html#green-power-devices

For info, ZGP devices can be paired directly on coordinator if you have a Silabs chip.

Ti chips, there is a must of a ZGP translator like a lamp or a plug

**Comment by nVenticDW** [comment]
Is the new Friends of Hue Switch X (FOH-2025-08) currently supported by Zigbee2MQTT?

Model on the rear plate FOH-2025-08
Model on rear label: PTM 215Z FOH (DB)
S3071-A215 FOH-7 DB

FCC-ID SZV_PTM215BZ

apparantly
top left AO         channel 15
bottom left A1   channel 20
top right B0       channel 11
bottom right B1 channel 25


### What happened?
Zigbee2MQTT opens the network correctly, but no Green Power commissioning frames, interview, or join event ever appear.

The log only shows:
- Network opened
- Network closed

No other activity is logged while attempting commissioning. 
and none of these comissioning sequences managed to be picked up in the debug log

### I was expecting
The switch should appear as a Green Power device or begin an interview.

### Zigbee2MQTT version
2.12.1

### Adapter
SLZB-06MU coordinator (Ember), firmware  v3.3.1

### Additional information  for commissioning
- Channel 11 tried
- Permit join enabled
- Other Zigbee devices work normally.
- Senic documentation only describes pairing with a Philips Hue Bridge.
-
- Tried the comissioning  sequences that I found in various sources:
a) Multi button  
 1. Top left
  2. Hold top right ~10 seconds
  3. Top right + bottom left
  4. Top left
b) 10 seconds top right
c) 10 seconds top left
d) 10 seconds bottom left
e) 10 seconds bottom right
f)  top right 7 seconds, top right bottom left together once, but apprantly I need a nearby green power proxy like philips hue bulb 
g) top right 7secs+, top right 1 sec, top right 7secs+, then close by Immediately press A1 and B0 simultaneously (Bottom Left + Top Right)  

I understand if b, c, d or e are seen in the logs they need to be closed off with bottom right and bottom left simulatainiously

Newxt I will try a hue bulb near the switch  but it isn't really clear

**Comment by nVenticDW** [comment]
The friends of Hue switch works . 


set SLZB to channel 11
carry out the button sequence with the rockers in place 

a) Top left   - to wake up the switch (this step might not be necessary)
b) Hold top right ~10 seconds  - to pair 
c) Top right + bottom left (this step is probably not necessary)
d) Top left  (this step is probably not necessary)

I will test again when I commission another switch, whether c) and d) are necessary
some sources say the rocker covers should be removed before pressing buttons- this is wrong
some sources say you need a Hue light in the network- this was NOT necesssary

**Comment by chris-1243** [comment]
> a) Top left - to wake up the switch (this step might not be necessary)

Not necessary.

> b) Hold top right ~10 seconds - to pair

Right

> c) Top right + bottom left (this step is probably not necessary)

This step is mandatory in order to complete the pairing sequence. Do not miss it.

> d) Top left (this step is probably not necessary)

Not necessary

Before doing annother pairing try. Remove your device, reset it, restart Z2M to remove anything which might be cached and pair it again via a specific router.
As you said, it doesn't have to be a HUE device. None of mine are paired via a HUE device. I would avoid Tuya. As far as I have tested, Innr, Ikea (not all devices), Ubysis, TI or Silabs coordinator flashed as router all work fine.
